src/libmongoc/doc/conf.py

```diff
@@ -29,6 +29,8 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 master_doc = 'index'
 
+intersphinx_disabled_reftypes = []
```

Comment:
Am I correct that the error is due to intersphinx including `std:doc` as a default in Sphinx 5.0: https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_disabled_reftypes

If so, I suggest noting this with a comment.

```suggestion
# Set an empty list of disabled reftypes.
# Sphinx 5.0 disables "std:doc" by default.
# Many documentation references use :doc:
intersphinx_disabled_reftypes = []
```